From 87fbe1ea65dd27f18fa257f66c4d30d45e64eed2 Mon Sep 17 00:00:00 2001
From: Hannes Reinecke <hare@suse.de>
Date: Mon, 11 Feb 2019 14:12:23 +0100
Subject: [PATCH] 71-nvme-iopolicy-netapp.rules: add default rules for NetApp
 Ontap

NetApp ONTAP controllers provide several paths to the same subsystem,
so we should be switching to 'round-robin' iopolicy to provide the
best performance.

Signed-off-by: Hannes Reinecke <hare@suse.com>
---
 scripts/71-nvme-iopolicy-netapp.rules | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 scripts/71-nvme-iopolicy-netapp.rules

diff --git a/scripts/71-nvme-iopolicy-netapp.rules b/scripts/71-nvme-iopolicy-netapp.rules
new file mode 100644
index 0000000..0d693a7
--- /dev/null
+++ b/scripts/71-nvme-iopolicy-netapp.rules
@@ -0,0 +1,2 @@
+# Enable round-robin for NetApp ONTAP
+ACTION=="add", SUBSYSTEM=="nvme-subsystem", ATTR{model}=="NetApp ONTAP Controller", ATTR{iopolicy}="round-robin"
-- 
2.13.7

